templates/home/_fsf_rust.html

```diff
@@ -10,8 +10,45 @@ <h4>Why are snaps good for Rust projects?</h4>
     </ul>
   </div>
 
+  <div class='col-6'>
+    <h4>How Parity defines snapcraft.yaml:</h4>
+    <code class="p-code-yaml"><pre>
+<b>name</b>: parity
+<b>version</b>: git
+<b>summary</b>: Fast, light, robust Ethereum implementation
+<b>description</b>: |
+  Parity's goal is to be the fastest, lightest, and 
+  most secure Ethereum client. We are developing 
+  Parity using the sophisticated and cutting-edge
+  Rust programming language. Parity is licensed 
+  under the GPLv3, and can be used for all your 
+  Ethereum needs.
+
+<b>confinement</b>: devmode
+<b>base</b>: core17
+
+<b>parts</b>:
+  <b>parity</b>:
+    <b>source</b>: .
+    <b>plugin</b>: rust
+    <b>build-attributes</b>: [no-system-libraries]
+    <b>build-packages</b>:
+      - libudev-dev
+      - libssl-dev
+      - make
+      - pkg-config
+    <b>stage-packages</b>:
+      - libssl1.0.0
+      - libudev1
+      - libstdc++6
+
+<b>apps</b>:
+  <b>parity</b>:
+    <b>command</b>: parity</pre></code>
+  </div>
+
```

Comment:
The snapcraft.yaml fails to build (even after correcting `base:` and dropping `no-system-libraries`:
```
error: failed to load source for a dependency on `parity-ui-old-precompiled`

Caused by:
  Unable to update https://github.com/paritytech/js-precompiled.git?branch=v1#f3fa5e5e

Caused by:
  revspec 'f3fa5e5efb0ed8ada6cd091694fcdaa921e6720f' not found; class=Reference (4); code=NotFound (-3)
Failed to run '/root/.cargo/bin/cargo +stable fetch --manifest-path /root/parts/parity/src/Cargo.toml' for 'parity': Exited with code 101.
Verify that the part is using the correct parameters and try again.
Run the same command again with --debug to shell into the environment if you wish to introspect this failure.
An error occurred when trying to execute 'sudo -i snapcraft snap' with 'multipass': returned exit code 2.
```